docs/api-usage.rst

```diff
@@ -192,6 +192,17 @@ You can print a Gitlab Object. For example:
    # Or explicitly via `pformat()`. This is equivalent to the above.
    print(project.pformat())
 
+You can also extend the object if the parameter isn't explicitly listed. For example,
+if you want to update a field that has been newly introduced to the Gitlab API, setting
+the value on the object is accepted:
+
+.. code-block:: python
+
+   issues = project.issies.list(state='opened')
```

Comment:
```suggestion
   issues = project.issues.list(state='opened')
```

Reply:
Well I feel dumb. But good catch, thank you.